.vscode/launch.json

```diff
@@ -37,22 +37,6 @@
       "internalConsoleOptions": "openOnSessionStart",
       "outputCapture": "std"
     },
-    {
```

Comment:
l: I never used these launch scripts so my logaf is practically nonexistent but is there a particular reason to remove them?

Reply:
I was more thinking why we would need them 😅